extmod/nimble/modbluetooth_nimble.c

```diff
@@ -52,38 +52,21 @@ STATIC int8_t ble_hs_err_to_errno_table[] = {
     [BLE_HS_EAGAIN] = MP_EAGAIN,
     [BLE_HS_EALREADY] = MP_EALREADY,
     [BLE_HS_EINVAL] = MP_EINVAL,
-    [BLE_HS_EMSGSIZE] = MP_EIO,
     [BLE_HS_ENOENT] = MP_ENOENT,
     [BLE_HS_ENOMEM] = MP_ENOMEM,
     [BLE_HS_ENOTCONN] = MP_ENOTCONN,
     [BLE_HS_ENOTSUP] = MP_EOPNOTSUPP,
-    [BLE_HS_EAPP] = MP_EIO,
-    [BLE_HS_EBADDATA] = MP_EIO,
-    [BLE_HS_EOS] = MP_EIO,
-    [BLE_HS_ECONTROLLER] = MP_EIO,
     [BLE_HS_ETIMEOUT] = MP_ETIMEDOUT,
-    [BLE_HS_EDONE] = MP_EIO,               // TODO: Maybe should be MP_EISCONN (connect uses this for "already connected").
```

Comment:
Perhaps this line should stay, otherwise the TODO is lost.

Reply:
Changed.